Makefile

```diff
@@ -0,0 +1,54 @@
+all: build test check-coverage lint  ## test, check coverage and lint
+	@if [ -e .git/rebase-merge ]; then git --no-pager log -1 --pretty='%h %s'; fi
+	@echo '$(COLOUR_GREEN)Success$(COLOUR_NORMAL)'
+
+clean::  ## Remove generated files
+
+.PHONY: all clean
+
+# --- Build and lint ---
```

Comment:
hip hip hooray. another form - three dashes and three dashes :-)

Reply:
yeah, that's the one i wanted... counting is hard. tell me what _you_ want and I will do it.

Reply:
I am completely unfussed apart from wanting something else. but that's my secret. this is fine.